app/assets/javascripts/auto_submit_form.js

```diff
@@ -5,7 +5,9 @@
 
   GOVUK.autoSubmitForm = {
     attach: function () {
-      $('form.js-auto-submit').submit();
+        var destinationLink = document.getElementById('saml-response-form');
```

Comment:
We now have a combination of jQuery and native API lookups here. Much though it pains me to suggest using jQuery over the native API, the following might be more consistent:

```suggestion
        var destinationLink = $('#saml-response-form')[0];
```

Also, this file seems to be using 2 space indentation, whereas this block is now indented by 4. Not sure if this is a problem or not. (I would personally applaud the switch to 4 space indenting... 😅 )